### PR TITLE
add kubernetes-client example for secret

### PR DESCRIPTION
**Pull request by Hugo-Ho**
## Description
add kubernetes-client example for secret

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

#3677

### CODE REVIEW

**Comment by rohanKanojia** [comment]
@Hugo-Ho : Thanks for your PR. Could you please add license header with `mvn -N license:format`?

**Comment by Hugo-Ho** [comment]
> @Hugo-Ho：感谢您的公关。你能添加许可证标题`mvn -N license:format`吗？

OK, I'll finish it

File: kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/SecretExamples.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      Map<String, String> data = new HashMap<>();
+      data.put("tls.crt", "YWFh");
+      data.put("tls.key", "YmJi");
+      final Secret secret = new SecretBuilder()
+        .withNewMetadata().withName(secretName).endMetadata()
+        .withType("kubernetes.io/tls")
+        .withImmutable(false)
+        .addToData(data)
+        .build();
+      Secret secretCreated = client.secrets().create(secret);
```
Comment (rohanKanojia) [inline kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/SecretExamples.java:59]: A Secret is a namespaced resource. Right now here client would pick whatever namespace is configured in kube config current context. Do you think we should provide namespace explicitly?
```suggestion
      Secret secretCreated = client.secrets()..inNamespace(namespace).create(secret);
```

**Review by oscerd — APPROVED** [review]

**Comment by rohanKanojia** [comment]
@Hugo-Ho: polite ping, still working on this?

**Review by rohanKanojia — APPROVED** [review]

**Review by manusa — APPROVED** [review]

**Pull request merged**